scripts/corelli/calibration/utils.py

```diff
@@ -24,10 +26,11 @@
 from Calibration.tube_calib_fit_params import TubeCalibFitParams
 
 # Functions exposed to the general user (public) API
-__all__ = ['apply_calibration', 'load_banks', 'calibrate_tube']
+__all__ = ['apply_calibration', 'preprocess_banks', 'load_banks', 'calibrate_tube']
 
 # Type aliases
-InputTable = Union[str, TableWorkspace]  # allowed types for the input calibration table to append_bank_number
+InputTable = Union[
+    str, TableWorkspace]  # allowed types for the input calibration table to append_bank_number
```

Comment:
?

Reply:
flake8 is complaining, so I use `yapf` to reformat the region it is complaining about.